WcaOnRails/app/models/user.rb

```diff
@@ -376,6 +376,10 @@ def can_manage_competition?(competition)
     can_admin_results? || competition.organizers.include?(self) || competition.delegates.include?(self)
   end
 
+  def can_view_competition_results?(competition)
+    can_admin_results?
```

Comment:
Shouldn't this method return true even if the user isn't able to admin results, in the case they are posted?